wrapping/itkTileMergeImageFilter.wrap

```diff
@@ -1,10 +1,13 @@
 itk_wrap_class("itk::TileMergeImageFilter" POINTER)
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     foreach(t ${WRAP_ITK_SCALAR})
-      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
+      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}, double")
     endforeach()
-    foreach(t ${WRAP_ITK_COLOR})
-      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
+    foreach(t ${WRAP_ITK_RGB})
+      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}, itk::RGBPixel<float>")
```

Comment:
Same as above -- `NumericTraits<RGBPixel<float>>::AccumulateType` seems to already be `RGBPixel<double>` from [`itkNumeritcTraitsRGBPixel.h`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkNumericTraitsRGBPixel.h#L62)

Reply:
RGB and RGBA pixels use [`RGBPixel<ElementAccumulateType>`](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc02/Modules/Core/Common/include/itkNumericTraitsRGBPixel.h#L62), which they get from the element/component type.

Reply:
Resolved above - some scalar types `t` for wrapping may be specialized with other `AccumulateType`s in their `NumericTraits` definitions.

Reply:
Before, RGBUC would use RGBUS for accumulation types. #196 suggests that is not enough in some cases. The main reason for this patch is to alleviate that by increasing accumulation type's bit-depth. And `float` should be enough, because Python wrappings only really have the option to expose 8-bit and 16-bit RGB/A components.

Reply:
Without explicitly setting the accumulation types, the defaults were used. It is surprising that those gray rectangles arise for relatively small images used by @zhusihan-python, but I confirmed the problem by reproducing it on my computer.